website/pages/home/img/partner-logos/gitlab-logo-gray-stacked-rgb.svg

```diff
@@ -0,0 +1 @@
+<svg enable-background="new 0 0 865 784" viewBox="0 0 865 784" xmlns="http://www.w3.org/2000/svg"><g fill="#8c929d"><path d="m439.246 496.833h-21.8l.1 162.5h88.3v-20.1h-66.5z"/><path d="m-387.454 487.533c-5.5 5.7-14.6 11.4-27 11.4-16.6 0-23.3-8.2-23.3-18.9 0-16.1 11.2-23.8 35-23.8 4.5 0 11.7.5 15.4 1.2v30.1zm-22.6-98.5c-17.6 0-33.8 6.2-46.4 16.7l7.7 13.4c8.9-5.2 19.8-10.4 35.5-10.4 17.9 0 25.8 9.2 25.8 24.6v7.9c-3.5-.7-10.7-1.2-15.1-1.2-38.2 0-57.6 13.4-57.6 41.4 0 25.1 15.4 37.7 38.7 37.7 15.7 0 30.8-7.2 36-18.9l4 15.9h15.4v-83.2c-.1-26.3-11.5-43.9-44-43.9z" transform="translate(977.32744 143.284396)"/><path d="m-418.154 499.833c-8.2 0-15.4-1-20.8-3.5v-67.3-7.8c7.4-6.2 16.6-10.7 28.3-10.7 21.1 0 29.2 14.9 29.2 39 0 34.2-13.1 50.3-36.7 50.3m9.2-110.6c-19.5 0-30 13.3-30 13.3v-21l-.1-27.8h-9.8-11.5l.1 158.5c10.7 4.5 25.3 6.9 41.2 6.9 40.7 0 60.3-26 60.3-70.9-.1-35.5-18.2-59-50.2-59" transform="translate(1099.766904 143.12893)"/><path d="m-382.154 370.833c19.3 0 31.8 6.4 39.9 12.9l9.4-16.3c-12.7-11.2-29.9-17.2-48.3-17.2-46.4 0-78.9 28.3-78.9 85.4 0 59.8 35.1 83.1 75.2 83.1 20.1 0 37.2-4.7 48.4-9.4l-.5-63.9v-7.5-12.6h-59.5v20.1h38l.5 48.5c-5 2.5-13.6 4.5-25.3 4.5-32.2 0-53.8-20.3-53.8-63-.1-43.5 22.2-64.6 54.9-64.6" transform="translate(584.042117 143.630796)"/><path d="m-438.154 354.233h-21.3l.1 27.3v11.2 6.5 11.4 65 .2c0 26.3 11.4 43.9 43.9 43.9 4.5 0 8.9-.4 13.1-1.2v-19.1c-3.1.5-6.4.7-9.9.7-17.9 0-25.8-9.2-25.8-24.6v-65h35.7v-17.8h-35.7z" transform="translate(793.569045 142.577463)"/><path d="m279.946 659.333h21.3v-124h-21.3z"/><path d="m279.946 518.233h21.3v-21.3h-21.3z"/></g><path d="m432.445 434.522 62.199-191.322h-124.307z" fill="#e24329"/><path d="m283.216 243.2-18.941 58.126c-1.714 5.278.137 11.104 4.661 14.394l163.509 118.801z" fill="#fca326"/><path d="m283.216 243.2h87.12l-37.494-115.224c-1.919-5.895-10.282-5.895-12.27 0z" fill="#e24329"/><path d="m581.764 243.2 18.873 58.126c1.714 5.278-.137 11.104-4.661 14.394l-163.531 118.802z" fill="#fca326"/><path d="m581.764 243.2h-87.12l37.425-115.224c1.919-5.895 10.282-5.895 12.269 0z" fill="#e24329"/><path d="m432.445 434.522 62.199-191.322h87.12z" fill="#fc6d26"/><path d="m432.445 434.522-149.229-191.322h87.121z" fill="#fc6d26"/></svg>
```

Comment:
Opted to add a local file here to match the pattern used by other logo assets for this grid.